### PR TITLE
Birdmeat from bird species. [NO GBP]

### PR DESCRIPTION
**Pull request by xXPawnStarrXx**
## About The Pull Request
This was a request that started at Teshari, I then pushed the boat out further by making Vox produce 'chicken' meat too.

## Why It's Good For The Game
People in. ***WHIRRR!*** delicious out.

## Proof Of Testing
image

## Changelog

:cl:
fix: prevented avian species from producing mamallian meat
/:cl:

### CODE REVIEW

**Comment by Redrover1760** [comment]
Warcrimes

**Comment by YakumoChen** [comment]
Putting my foot down since I'll need to go back and fix these

This needs to be made into a child meat item with the GORE food flag added, including its component steak/etc foods, to properly put it in line with eating other "man-made" meats.

**Comment by YakumoChen** [comment]
The gore flag is for cannibal meats, you're thinking of the RAW tag.

**Comment by xXPawnStarrXx** [comment]
> The GORE flag is for cannibal meats, you're thinking of the RAW flag

image
Corgi aren't cannibal food, nor are mice, but they still have the gore flag. It's seeming chaotic and random which ones get the flag.

**Comment by YakumoChen** [comment]
https://github.com/Bubberstation/Bubberstation/blob/97a681fbc9416d9b049b907e08faa32952ef3182/code/game/objects/items/food/meatslab.dm#L57

Compare to a generic meat slab above

GORE tag is reserved for meat a standard human would consider gross or unethical, ie dog meat, cannibalism, rodent meat

**Comment by YakumoChen** [comment]
> Chicken itself at base doesn't even have the gore flag, it seems to carry all of it's traits such as when you examine it, it states whom it was made from.

Have you tested if this applies to food made from the meat slab?
I believe human meat has recipes specifically coded to support the naming behaviour

File: modular_skyrat/modules/better_vox/code/vox_species.dm [hunk 1]
```diff
@@ -16,6 +16,7 @@
 	mutantbrain = /obj/item/organ/internal/brain/vox
 	breathid = "n2"
 	mutant_bodyparts = list()
+	meat = /obj/item/food/meat/slab/chicken
```
Comment (YakumoChen) [inline modular_skyrat/modules/better_vox/code/vox_species.dm:19]: Need to update the species still

**Comment by xXPawnStarrXx** [comment]
> Have you tested if this applies to food made from the meat slab? I believe human meat has recipes specifically coded to support the naming behaviour

Default human:
image

Teshari patty:
image

It won't end up with different named burgers and chicken, but that's intentional. I want them to be nameless.

**Comment by theselfish** [comment]
I already done most of this upstream.
https://github.com/Skyrat-SS13/Skyrat-tg/pull/26021
Sans the gore thing.

**Comment by xXPawnStarrXx** [comment]
> I already done most of this upstream. [Skyrat-SS13/Skyrat-tg#26021](https://github.com/Skyrat-SS13/Skyrat-tg/pull/26021) Sans the gore thing.

Ah, so it's just waiting on it to be merged here.

**Pull request closed without merging**